sdk/core/test/integration/otp/index.test.ts

```diff
@@ -311,6 +312,25 @@ describe('CommonNetworkMember (flows that require OTP)', () => {
     expect(responseError.name).to.eql('COR-17')
   })
 
+  it('#passwordlessLogin with custom messages ', async () => {
+    const cognitoUsername = COGNITO_USERNAME
+    const stmp = `${Date.now()} stmp`
+    const messageParameters: MessageParameters = {
+      message: `Your verification code is ${stmp} {{CODE}}.`,
+      subject: `${stmp} {{CODE}}.`,
+    }
+    const fullOptions = getOptionsForEnvironment(true)
+
+    const token = await CommonNetworkMember.passwordlessLogin(cognitoUsername, fullOptions, messageParameters)
+
+    // NOTE: wait for 180s before providing the answer
```

Comment:
Why do we need to wait for 3 minutes?

Reply:
it is a copy past of Otp from other tests. I think without delay otp helper give unreliable results

Reply:
@Volland so `DELAY` value is 1000, this means that its will wait 1s and not 180s, so not really clear why you added such comment, please remove it. Also waiting in tests 3 minutes its toooooo long, and could not be accepted